src/Symfony/Bundle/FrameworkBundle/Tests/Test/WebTestCaseTest.php

```diff
@@ -239,15 +239,15 @@ public function testAssertCheckboxChecked()
     {
         $this->getCrawlerTester(new Crawler('<html><body><form><input type="checkbox" name="rememberMe" checked>'))->assertCheckboxChecked('rememberMe');
```

Comment:
i would run all of the tests in this class with and without a html5 doctype.
so with:
1. `<html>` and
2. `<!DOCTYPE html>`

this will expose the previous bug and prevent regression

Reply:
I tested against HTML5 in `testAssertCheckboxChecked` and `testAssertCheckboxNotChecked`. Do you really think it should be done for every other test (they all pass)?

Reply:
well it's debatable. we're testing the 2 different parses in the DomCrawler Crawler.
There was an unexposed bug here so it might make sense to test every assertion againsts both parsers.
I'm indifferent on adding it to every test

Maybe someone else has a stronger opinion.

Reply:
@bendavies i agree with you, we should test all assertions with and without a html5 doctype. Perhaps add new tests for checked and not checked with doctype or at least add tests for the error cases.

Reply:
FTR tests were added for error cases. Waiting for a core member incentive to add them in all others.

Reply:
doing that for all other tests not related to this bugfix could be done in a separate PR